subprojects/docs/src/docs/release/notes.md

```diff
@@ -52,9 +52,13 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
-### Tooling API deprecation
+### Deprecation of old Tooling API version 
+
+The following supports are deprecated now and will be removed in Gradle 5.0. You should avoid using them:
+
+- Running Gradle via Tooling API older than 3.0
```

Comment:
⭕️ This has not been implemented yet, consider moving it to your next PR.

Reply:
Sure. Thanks.